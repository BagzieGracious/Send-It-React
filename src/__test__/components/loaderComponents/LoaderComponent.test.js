import React from 'react';
import { shallow } from 'enzyme';
import LoaderComponent from '../../../components/loaderComponent';

describe('LoaderComponent test', () => {
	it('should render without crashing', () => {
		const wrapper = shallow(<LoaderComponent />);
		expect(wrapper).toMatchSnapshot();
	});
});
