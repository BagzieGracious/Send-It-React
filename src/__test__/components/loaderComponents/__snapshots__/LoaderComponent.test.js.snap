// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderComponent test should render without crashing 1`] = `ShallowWrapper {}`;
